discord/ext/commands/context.py

```diff
@@ -348,7 +195,7 @@ async def send_help(self, *args: Any) -> Any:
         Any
             The result of the help command, if any.
         """
-        from .core import Command, Group, wrap_callback
```

Comment:
Removing the import for `Group` here is causing the unit tests workflow to fail.

Reply:
oh